lxd/storage/backend_lxd.go

```diff
@@ -57,7 +57,7 @@ func (b *lxdBackend) MigrationTypes(contentType drivers.ContentType) []migration
 }
 
 // create creates the storage pool layout on the storage device.
-func (b *lxdBackend) create(dbPool *api.StoragePool, op *operations.Operation) error {
+func (b *lxdBackend) create(dbPool *api.StoragePoolsPost, localOnly bool, op *operations.Operation) error {
```

Comment:
@stgraber could you add a comment to the function's description indicating what `localOnly` does please? From what I can make out, it only creates the local directory structures and doesn't call out to the driver to do the create, presumably this is used when adding an existing remote storage to a pool.